cylc/flow/wallclock.py

```diff
@@ -162,17 +157,19 @@ def get_time_string(date_time, display_sub_seconds=False,
         time_zone_string = TIME_ZONE_STRING_UTC
         if date_time_is_local:
             date_time = date_time - timedelta(
-                hours=TIME_ZONE_LOCAL_UTC_OFFSET_HOURS,
-                minutes=TIME_ZONE_LOCAL_UTC_OFFSET_MINUTES
+                hours=get_local_time_zone()[0],
+                minutes=get_local_time_zone()[1]
```

Comment:
`get_local_time_zone()` ought to be called only once at the start of this function.